R/tm_a_pca.R

```diff
@@ -282,7 +282,6 @@ ui_a_pca <- function(id, ...) {
         )
       ),
       forms = tagList(
```

Comment:
Do we need `taglist()` now ?

Reply:
replied the same as in the tmc, I think this is fine and shows the ability for this part to be extendable